composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewModel.kt

```diff
@@ -12,33 +12,57 @@ class TimerViewModel : ViewModel() {
     private val _loading = MutableStateFlow(false)
     val loading = _loading.asStateFlow()
 
-    private val _exercises = MutableStateFlow(listOf("Lagartijas","gallitos", "dominadas"," sentadillas"))
+    private val _exercises =
+        MutableStateFlow(listOf("Lagartijas", "gallitos", "dominadas", " sentadillas"))
     val exercises = _exercises.asStateFlow()
 
+    private val _reps = MutableStateFlow(0)
+    val reps = _reps.asStateFlow()
+
     fun onEvent(event: TimerViewEvent) {
         when (event) {
-            is TimerViewEvent.ClickAdd -> {
+            is TimerViewEvent.ClickedAdd -> {
                 viewModelScope.launch {
-                    ClickAdd()
-                }}
-            is TimerViewEvent.onDelete -> {
-                onDelete(event.item)
+                    onClickedAdd()
                 }
+            }
 
-    }
+            is TimerViewEvent.ClickedDelete -> {
+                onClickedDelete(event.index)
+            }
+
+            is TimerViewEvent.AddReps -> {
+                addReps()
+            }
+
+            is TimerViewEvent.RemoveReps -> {
+                removeReps()
+            }
```

Comment:
```suggestion
            is TimerViewEvent.ClickedDelete -> onClickedDelete(event.index)
            is TimerViewEvent.AddReps -> addReps()
            is TimerViewEvent.RemoveReps -> removeReps()
```

1000000% subjective feedback that I'd love to see this written like this